bps/resources/LSSTCam-imSim/DRP-DP0.2.yaml

```diff
@@ -10,10 +10,6 @@
 #
 pipetask:
   # DRP tasks
-  makeDirectWarp:
-    requestMemory: 8192
-  makePsfMatchedWarp:
-    requestMemory: 8192
```

Comment:
I don't see a corresponding addition of makeWarp here. Does it fit it in the default memory request?

Reply:
The history here is that prior to the ticket that I'm mostly reverting, there was only a configuration of `makeWarp` with these values, and that ticket replaced it with configuration for `makeDirectWarp` and `makePsfMatchedWarp` (instead of adding those).  But I think there didn't need to be an override for any of these, because they really should fit in the default memory request, and that's consistent with the lack of memory overrides in `test-med-1.yaml` here.  I'm pretty sure there was just a one-off problem in the DP0.2 era that has long since been fixed.